components/tools/OmeroPy/src/omero/plugins/fs.py

```diff
@@ -435,6 +444,9 @@ def rename(self, args):
 template. By default the original files and import log are also
 moved.
 """
+        if platform.system() == 'Windows':
```

Comment:
@ximenesuk that is true for the top level check but within a class you can access, see https://github.com/openmicroscopy/openmicroscopy/pull/4565/files#diff-ad256e5e06d156adf521e47a4cdd8c37R68.

My comment was only suggestion, no need to be changed. I am planning to make `_isWindows` public that we will be able to use it in init as well (next PR once all of these warnings are merged).